gr-digital/lib/costas_loop_cc_impl.cc

```diff
@@ -204,6 +224,189 @@ int costas_loop_cc_impl::work(int noutput_items,
     return noutput_items;
 }
 
+int costas_loop_cc_impl::work2(int noutput_items,
```

Comment:
I really like the performance increase. However, for a number of reasons -- cross-compiling, runtime platform detection, ease of debugging, etc. -- we usually put platform-specific optimizations into Volk and call out to them from the block's work function. Is it feasible to export the fast FMA (and generic) core of this block to a Volk kernel?